packages/apps-config/src/api/spec/zenlink.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright 2017-2021 @polkadot/apps-config authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+export default {
+  AssetId: {
```

Comment:
I would just suggest adding the `Address` and `LookupSource` types explicitly. The reason is that the defaults are changing on the API layer, see https://github.com/polkadot-js/api/releases/tag/v3.5.1

(One of these types https://polkadot.js.org/docs/api/FAQ#i-cannot-send-transactions-sending-yields-address-decoding-failures)